src/MicrosoftTeams.schema.json

```diff
@@ -57,22 +57,36 @@
         },
         "name": {
             "type": "string",
-            "description": "The display name for the package.",
-            "maxLength": 16
+            "description": "The short and long display name for the app.",
+            "properties": {
+                "short": {
+                    "type": "string",
+                    "description": "A short display name for the app",
+                    "maxLength": 50
+                },
+                "full": {
+                    "type": "string",
+                    "description": "The full name of the app, used if the full app name exceeds 50 characters",
+                    "maxLength": 100
+                }
+            },
+            "required": [
+                "short"
```

Comment:
Many app names will fully fit within the 50 character limit. Seems weird to require it in that case.